index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <nav class="nav-bar">
+      <h2>John Omachi</h2>
+      <img src="./images/hamburger.png" alt="" />
+    </nav>
+    <section class="jumbotron">
+      <h1>John Omachi</h1>
+      <h2>Software Developer</h2>
+      <p>
+        I can help you build a product , feature or website Look through some of
+        my work and experience! If you like what you see and have a project you
+        need coded, don’t hestiate to contact me.
+      </p>
+    </section>
+    <div class="prog-lang-skills">
+      <div class="prog-lan">
+        <ul>
+          <li class="list-head">Language</li>
+          <li>Javascript</li>
+          <li>Ruby</li>
+          <li>Html</li>
+          <li>Css</li>
+        </ul>
+        <ul>
+          <li class="list-head">Frameworks</li>
+          <li>Bootstrap</li>
+          <li>Ruby on Rail</li>
+          <li>RSPec</li>
+          <li>CapyBara</li>
+          <li>Selenium</li>
+        </ul>
+        <ul>
+          <li class="list-head">Skills</li>
+          <li>Codekit</li>
+          <li>Github</li>
+          <li>Codepen</li>
+          <li>Gitlab</li>
+          <li>Terminal</li>
+        </ul>
+      </div>
+    </div>
+
+    <div class="social-media">
+      <ul>
+        <li>
+          <a href="#">
+            <img src="./images/github.png" alt="" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./images/Vector.png" alt="" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./images/Vector2.png" alt="" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./images/twitter.png" alt="" />
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <img src="./images/Vector3.png" alt="" />
+          </a>
+        </li>
+      </ul>
+    </div>
```

Comment:
- [ ] `required` kindly align the social media icons like the figma template please see screenshots below

Figma template
![Screenshot from 2022-06-28 20-21-08](https://user-images.githubusercontent.com/66230306/176567406-689de772-bc67-46db-9ff2-a76dbed42fca.png)

Your template
![Screenshot from 2022-06-28 20-25-48](https://user-images.githubusercontent.com/66230306/176567543-cf77c959-0de8-4f5e-8967-162e4746350e.png)

Reply:
Social media icons have been repositioned successfully.
[@mwanawabangona](https://github.com/mwanawabangona)

Reply:
![image](https://user-images.githubusercontent.com/97337740/176582916-08e214f9-1160-4f27-bf22-3b5d3cb0db36.png)